doc/bus.rst

```diff
@@ -59,6 +59,14 @@ Filtering
 Message filtering can be set up for each bus. Where the interface supports it, this is carried
 out in the hardware or kernel layer - not in Python.
 
+Example defining two filters, one to pass 11-bit ID 0x451, the other to pass 29-bit ID 0xA0000:
+
+.. code-block:: python
+
+    filters = [{"can_id": 0x451, "can_mask":0x7FF,"extended":False},{"can_id": 0xA0000, "can_mask":0x1FFFFFFF,"extended":True}]
+    bus = can.interface.Bus(channel='can0', bustype='socketcan_native',can_filters=filters)
```

Comment:
`socketcan_native` and `socketcan_ctypes` are deprecated, please see https://python-can.readthedocs.io/en/master/interfaces/socketcan.html.
Please use `socketcan` interface.